### PR TITLE
backupccl: disable span configs for full cluster restore jobs test

### PR DESCRIPTION
**Pull request by RichardJCai**
Release note: None

Related https://github.com/cockroachdb/cockroach/issues/75216

### CODE REVIEW

File: pkg/ccl/backupccl/full_cluster_backup_restore_test.go [hunk 1]
```diff
@@ -303,6 +303,7 @@ CREATE TABLE data2.foo (a int);
 	})
 
 	t.Run("ensure that jobs are restored", func(t *testing.T) {
+		skip.WithIssue(t, 75216, "flaky under testrace")
```
Comment (irfansharif) [inline pkg/ccl/backupccl/full_cluster_backup_restore_test.go:306]:
```diff
diff --git i/pkg/ccl/backupccl/full_cluster_backup_restore_test.go w/pkg/ccl/backupccl/full_cluster_backup_restore_test.go
index 5791a2b1be..941f5f51ec 100644
--- i/pkg/ccl/backupccl/full_cluster_backup_restore_test.go
+++ w/pkg/ccl/backupccl/full_cluster_backup_restore_test.go
@@ -48,6 +48,7 @@ func TestFullClusterBackup(t *testing.T) {

        params := base.TestClusterArgs{
                ServerArgs: base.TestServerArgs{
+                      DisableSpanConfigs: true, // TODO(irfansharif): #75060.
                        Knobs: base.TestingKnobs{
                                SpanConfig: &spanconfig.TestingKnobs{
                                        // We compare job progress before and after a restore. Disable
```

Mind using diff instead?

Comment (irfansharif) [reply pkg/ccl/backupccl/full_cluster_backup_restore_test.go:306]: And updating the commit message/PR title accordingly. Pretty sure it's fallout from #73876 that I have yet to investigate.

Comment (RichardJCai) [reply pkg/ccl/backupccl/full_cluster_backup_restore_test.go:306]: Wait I don't see a `DisableSpanConfigs` field under `TestServerArgs`

Comment (RichardJCai) [reply pkg/ccl/backupccl/full_cluster_backup_restore_test.go:319]: Updated

**Review by irfansharif — APPROVED** [review]
Thanks Richard!

**Comment by irfansharif** [comment]
Just going to bors it for ya to maybe clear up CI.

bors r+

**Pull request merged**